src/fastoad/notebooks/02_explanation/01_summary.ipynb

```diff
@@ -0,0 +1,226 @@
+{
```

Comment:
<p>"The fifth notebooks" actually links to the forth.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Changed in dfe233f7